app/redux-modules/feedback.js

```diff
@@ -14,14 +15,25 @@ const UPDATE_FEEDBACK = 'feedback/UPDATE_FEEDBACK';
 const initialNavState = {
   daily: {},
   weekly: {},
-  list: {}
+  list: {},
+  dailySynced: false,
+  weeklySynced: false
 };
 
 export default function reducer(state = initialNavState, action) {
   switch (action.type) {
-    case GET_QUESTIONS: {
+    case GET_FEEDBACK_QUESTIONS_COMMIT: {
```

Comment:
We need to add a case for the reducer in the `GET_FEEDBACK_QUESTIONS_REQUEST` to set the synced flag as false.